rectools/models/sasrec.py

```diff
@@ -1137,14 +1137,16 @@ def __init__(  # pylint: disable=too-many-arguments, too-many-locals
         epochs: int = 3,
         verbose: int = 0,
         deterministic: bool = False,
-        cpu_n_threads: int = 0,
+        recommend_device: str = "cpu",
```

Comment:
default value should be "auto".
And type should be exactly the one that is accepted in trainer accelerator, not string

Reply:
changed to "auto"